WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelCustomPackageForm.swift

```diff
@@ -0,0 +1,142 @@
+import SwiftUI
+
+/// Form to create a new custom package to use with shipping labels.
+struct ShippingLabelCustomPackageForm: View {
+    @ObservedObject private var viewModel = ShippingLabelCustomPackageFormViewModel()
+    private let safeAreaInsets: EdgeInsets
+
+    init(safeAreaInsets: EdgeInsets) {
+        self.safeAreaInsets = safeAreaInsets
+    }
+
+    var body: some View {
+            VStack {
```

Comment:
Nit: As I see on Figma it seems that the spaces between sections are a bit larger than what we have here. I think it'd be nice if a specific spacing is defined; and something like 16 points can be reasonable for spacing between sections. WDYT?